docs/comparisons.md

```diff
@@ -80,10 +80,7 @@ JOIN raw.event_dims d
 -- must specify the is_incremental flag because this predicate will fail if the model has never run before
 {% if is_incremental() %}
     -- this filter dynamically scans the current model to find the date boundary
-    AND d.ds >= (SELECT MAX(ds) FROM {{ this }})
-{% endif %}
-{% if is_incremental() %}
-  WHERE e.ds >= (SELECT MAX(ds) FROM {{ this }})
+  WHERE d.ds >= (SELECT MAX(ds) FROM {{ this }})
```

Comment:
I believe the intent here was to show that if you wanted to apply the filter to both tables in the join then you would need to define this filter twice. This reduction removes the filter on `e` which means you would then do a full table scan on it.

Reply:
well shit you're right. that is what speed reading gets me